src/bolo-do-caco.md

```diff
@@ -0,0 +1,47 @@
+# Bolo do Caco _(Caco Bread)_
+
+_A tradicional Madeiran-Portuguese Bread_
+
+- ⏲️ Prep time: 1 hour
+- 🍳 Cook time: 45 min
+- 💤 Rest time: 3 days
+- 🍽️ Servings: 10 breads
+
+## Ingredients
+
+__Imperial__
+
+- 1¾ lb sweet potato
+- 8 cups all-purpose flour
+- ⅔ cup water (warm at 75F / 35C)
+- 3 tablespoons active dry - yeast
+- 3 teaspoons salt
+
+__Metric__
+- 750 g sweet potato
+- 1 kg all-purpose flour
+- 150 ml water (warm at 75F / 35C)
+- 3 tablespoons active dry yeast
+- 3 teaspoons salt
```

Comment:
Don't include salt and pepper and other ubiquitous things in the ingredients list.

Reply:
Hello, do you need me to remove it?